server/controllers/helperFunctions/tokenHelper.js

```diff
@@ -0,0 +1,23 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+/**
+ * All things pertaining to signing and verifying token
+ */
+class TokenHelper {
```

Comment:
please modify your class name. you can remove the "helper" since you are already in the helpers folder.
Also, you may want to consider using functions instead of a class